templates/case/_case_show/_lease_information/resident_complaint_board_case.html.twig

```diff
@@ -3,7 +3,7 @@
 <div class="form-row">
     <div class="form-group col-md-3">
         <label for="inputBasicInformation4">{% trans %}Lease started{% endtrans %}</label>
-        <input type="text" class="form-control" id="inputBasicInformation4" value="{{ case.leaseStarted|date(format_date) }}" readonly>
+        <input type="text" class="form-control" id="inputBasicInformation4" value="{{ case.leaseStarted is empty ? "" : case.leaseStarted|date(format_date) }}" readonly>
```

Comment:
You can use `{{ case.leaseStarted ? case.leaseStarted|date(format_date) }}` here (cf. https://twig.symfony.com/doc/3.x/templates.html#other-operators). Twig is clever and has some nice features.

Maybe we should consider adding a `date_nullable` filter that returns the empty string if the date is empty.